src/elements/fields/PhoneField/index.tsx

```diff
@@ -0,0 +1,324 @@
+      {fieldLabel}
+      <div
+        css={{
+          position: 'relative',
+          width: '100%',
+          whiteSpace: 'nowrap',
+          overflowX: 'hidden',
+          ...applyStyles.getTarget('sub-fc')
+        }}
+      >
+        <div
+          css={{
+            position: 'absolute',
+            cursor: 'pointer',
+            height: '100%',
+            transition: '0.2s ease all',
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+            borderRight: '1px solid #e6e6e6',
+            ...applyStyles.getTarget('fieldToggle'),
+            '&:hover': {
+              backgroundColor: '#e6e6e6'
+            }
+          }}
+          ref={triggerRef}
+          onClick={() => setShow(!show)}
+        >
+          {countryMap[curCountryCode].flag}
+          <DropdownArrow css={{ marginLeft: '3px' }} />
+        </div>
+        <Overlay
+          target={triggerRef.current}
+          show={show}
+          onHide={() => setShow(false)}
+          placement='bottom-start'
+        >
+          {(props) => {
+            ['placement', 'arrowProps', 'show', 'popper'].forEach(
+              (prop) => delete props[prop]
+            );
+            return (
+              <ul
+                css={{
+                  listStyleType: 'none',
+                  padding: 0,
+                  margin: 0,
+                  backgroundColor: 'white',
+                  cursor: 'pointer',
+                  boxShadow: '0 0 4px rgb(0 0 0 / 15%)',
+                  maxHeight: '210px',
+                  overflowY: 'scroll',
+                  overflowX: 'hidden',
+                  width: '400px'
+                }}
+                {...props}
+                ref={(ref) => {
+                  dropdownRef.current = ref;
+                  props.ref(ref);
+                }}
+              >
+                {supportedCountries}
+              </ul>
+            );
+          }}
+        </Overlay>
+        <input
+          id={servar.key}
+          css={{
+            height: '100%',
+            width: '100%',
+            ...bootstrapStyles,
+            ...applyStyles.getTarget('field'),
+            ...(inlineError ? { borderColor: ERROR_COLOR } : {}),
+            '&:focus': applyStyles.getTarget('active'),
+            '&:hover': applyStyles.getTarget('hover'),
+            '&:not(:focus)':
+              formattedNumber || !placeholder
+                ? {}
+                : { color: 'transparent !important' }
+          }}
+          required={required}
+          autoComplete={servar.metadata.autocomplete || 'on'}
+          placeholder=''
+          value={formattedNumber}
+          ref={(ref) => {
+            inputRef.current = ref;
+            setRef(ref);
+          }}
+          type='tel'
+          onBlur={() => setTriggerOnChange(!triggerOnChange)}
+          onKeyDown={(e) =>
+            e.key === 'Enter' && setTriggerOnChange(!triggerOnChange)
```

Comment:
I think it would be good UX to have it work this way: if focus is the phone number input, then if the user types + or any letter, the overlay opens and the input is in the search/filter  area.  The user uses up/down arrow to make a country choice, hits enter and then focus is on the input field to accept numeric input.  Having to go between mouse and keyboard is clunky.